torch/_torch_docs.py

```diff
@@ -736,11 +736,16 @@ def parse_kwargs(desc):
 
 add_docstr(torch.cat,
            r"""
-cat(seq, dim=0, out=None) -> Tensor
+cat(seq, dim=0, pad=False, pad_value=0, out=None) -> Tensor
```

Comment:
For a more straightforward API, how about make `pad_value` nullable and pad only it is not `None`

Reply:
cc @gchanan

Reply:
IIRC, we do have nullable integer support in jit now. cc @wanchaol

Reply:
Oh, you must be referring to this: https://github.com/pytorch/pytorch/pull/9596.

I guess this could work, although it precludes NaN from being a pad value.  That's not an issue with clamp because NaN as a min or max is bogus.  But that's not a big deal I guess.

Reply:
This is a terrible hack for clamp, which we shouldn't depend on. Adding support for `optional<T>` to the JIT shouldn't be too hard.